elixir/apps/web/lib/web/live/policies/components.ex

```diff
@@ -652,4 +652,28 @@ defmodule Web.Policies.Components do
     Domain.Policies.Condition.Changeset.valid_operators_for_property(property)
     |> Enum.map(&{condition_operator_option_name(&1), &1})
   end
+
+  def options_form(assigns) do
+    ~H"""
+    <fieldset class="flex flex-col gap-2 mt-4">
+      <.inputs_for :let={options} field={@form[:options]}>
+        <div class="flex items-center justify-between">
+          <legend class="text-xl mb-2 text-neutral-900">Options</legend>
+        </div>
+
+        <div>
+          <fieldset class="mb-2">
+            <.input
+              type="checkbox"
+              label="Allow users to access the internet outside of Firezone"
```

Comment:
This comment isn't so much around function, but more around user experience.  While I was trying to create a Policy for the Internet Resource, I saw this checkbox and wasn't actually sure what it was trying to convey.

Is this saying that users have the ability to stay signed in to the Firezone client to allow them to access private resources but bypass the Internet resource?

Reply:
cc @jamilbk that is what we discussed, this feature is really ambiguous, need much better content around it :(

Reply:
Hm yeah we may need a longer description and / or rewording here. Or we could keep this enabled behind the scenes for now and expose the option to disable it when we hear that requirement from customers. I'll give it some more thought tomorrow.

Reply:
I'd address disabling when we implement PAM

Reply:
Do we have any customer feedback around this feature?  Has any customer explicitly asked one way or another for the enable/disable feature?

Reply:
Yeah so far they've just wanted to allow enabling/disabling, but I believe it has been only admins asking for the feature for their technical team (DevOps).

For the broader workforce, calls from longer ago had expressed a need for this for Internet security for their workforce, but if we had to pick one or the other I'd say we should go with `true` here.

The other complication here is that, depending on the Policy that matches against your actor based which group you're in - you might be shown this flag or might not. The portal would need a deterministic way to select policies (perhaps based on smaller group first).

Based on the above I'm reluctant to expose this option and instead just default it to `true` for now and hide the UI.

WDYGT?

Reply:
Yeah, I agree.  I think setting the default to `true` and not exposing the option right now is probably the way to go.

Reply:
@AndrewDryga bump